src/classes/MyHandler/MyHandler.ts

```diff
@@ -1882,6 +1907,15 @@ export default class MyHandler extends Handler {
                 highValue: isContainsHighValue ? highValueMeta : undefined
             };
 
+            // don't use business logic if the bot is not operational
+            if (this.bot.isHalted) {
+                return {
+                    action: 'skip',
+                    reason: 'REVIEW',
+                    meta: meta
+                };
```

Comment:
We have the reason REVIEW here, but it does nothing because this directory wasn't touched.
https://github.com/TF2Autobot/tf2autobot/tree/a0ce7cbeb25497084235c071f6927dbb470ee7b8/src/classes/MyHandler/offer/review